ddtrace/debugging/_debugger.py

```diff
@@ -308,31 +306,39 @@ def _dd_debugger_wrapper(self, wrappers):
 
         def _(wrapped, args, kwargs):
             # type: (FunctionType, Tuple[Any], Dict[str,Any]) -> Any
-            if not any(probe.active for probe in wrappers.values()):
+            active_probes = [probe for probe in wrappers.values() if probe.active]
+
+            if not active_probes:
```

Comment:
I thought it would be more efficient as in the old implementation we go over the list twice. I would say maybe we should avoid the "disabled" stuff all together? I'm not sure this been used anymore.